test/studies/lulesh/intelVectBug/lulesh.compopts

```diff
@@ -0,0 +1,6 @@
+-M../../../release/examples/benchmarks/lulesh --fast
```

Comment:
I could, though that would just make the future pass and we could add it at that point...  Branch predicting, I think that we'd like to always tie vectorize to fast and that if vectorize is problematic in other ways, we'll fix those problems...  It was only the short window before the release that caused me to consider decoupling them.